plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -255,6 +255,13 @@ func init() {
 		},
 	})
 	addControllerRole(rbac.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + "cloud-provider"},
+		Rules: []rbac.PolicyRule{
+			rbac.NewRule("add", "get", "list", "watch").Groups(legacyGroup).Resources("configmaps").RuleOrDie(),
```

Comment:
Agreed. Seems to me that this would be better in `namespace_policy.go` and binded to the single namespace.